libs/wingsdk/src/target-tf-aws/service.ts

```diff
@@ -0,0 +1,338 @@
+            {
+              "Effect": "Allow",
+              "Action": [
+                "logs:CreateLogStream",
+                "logs:PutLogEvents",
+                "logs:CreateLogGroup"
+              ],
+              "Resource": "*"
+            },
+            {
+              "Effect": "Allow",
+              "Action": [
+                "ecr:BatchGetImage",
+                "ecr:GetDownloadUrlForLayer",
+                "ecr:GetAuthorizationToken"
+              ],
+              "Resource": "*"
+            }
+          ],
+        }),
+      }]
+    });
+
+    let taskRole = new IamRole(this, "TaskRole", {
+      assumeRolePolicy: JSON.stringify({
+        Version: "2012-10-17",
+        Statement: [
+          {
+            Effect: "Allow",
+            Principal: {
+              Service: "ecs-tasks.amazonaws.com",
+            },
+            Action: "sts:AssumeRole",
+          },
+        ],
+      }),
+      inlinePolicy: [{
+        name: `${this.assetName}-inline-policy`,
+        policy: Lazy.stringValue({
+          produce: () => {
+            this.policyStatments = this.policyStatments ?? [];
+            
+            if (this.policyStatments.length !== 0) {
+              return JSON.stringify({
+                Version: "2012-10-17",
+                Statement: this.policyStatments
+              });
+            }
+
+            return JSON.stringify({
+              Version: "2012-10-17",
+              Statement: [
+                {
+                  Effect: "Allow",
+                  Action: "none:null",
+                  Resource: "*"
+                }
+              ]
+            })
+          }
+        }),
+      }]
+    });
+
+    let logGroup = new CloudwatchLogGroup(this, "LogGroup", {
+      name: `/ecs/${this.assetName}`,
+      retentionInDays: 7,
+    });
+
+    let taskDef = new EcsTaskDefinition(this, "Task", {
+      family: this.assetName,
+      executionRoleArn: executionRole.arn,
+      taskRoleArn: taskRole.arn,
+      containerDefinitions: Lazy.anyValue({
+        produce: () => JSON.stringify([
+            {
+              name: this.assetName,
+              image: image.name,
+              environment: Object.entries(this._env).map(([key, value]) => {
+                return {
+                  name: key,
+                  value: value,
+                };
+              }
+              ),
+              logConfiguration: {
+                logDriver: "awslogs",
+                options: {
+                  "awslogs-group": `/ecs/${this.assetName}`,
+                  "awslogs-region": "us-east-1", // TODO: Can I ignore this?
+                  "awslogs-stream-prefix": logGroup.name,
+                }
+              }
+          }
+        ])
+      }) as any,
+      networkMode: "awsvpc",
+      requiresCompatibilities: ["FARGATE"],
+      memory: "512",
+      cpu: "256",
+    });
+
+    let subnetIds = app.subnets["private"].map((subnet) => subnet.id);
```

Comment:
Would it make sense to add `app.privateSubnets`? The `"private"` magic string is risky.

Reply:
Yea it probably does make sense, we have this magic string in other places as well so Ill update it everywhere.